test/integration/authorization_test.go

```diff
@@ -860,33 +1051,39 @@ func TestAuthorizationSubjectAccessReview(t *testing.T) {
 			User:   "danny",
 			Action: authorizationapi.Action{Verb: "get", Resource: "projects"},
 		},
+		kubeAuthInterface: clusterAdminLocalSARGetter,
 		response: authorizationapi.SubjectAccessReviewResponse{
 			Allowed:   true,
 			Reason:    "allowed by rule in default",
 			Namespace: "default",
 		},
 	}.run(t)
 	subjectAccessReviewTest{
-		description:      "cluster admin told danny cannot get projects cluster-wide",
-		clusterInterface: clusterAdminClient.SubjectAccessReviews(),
-		clusterReview:    askCanDannyGetProject,
+		description:       "cluster admin told danny cannot get projects cluster-wide",
+		clusterInterface:  clusterAdminClient.SubjectAccessReviews(),
+		clusterReview:     askCanDannyGetProject,
+		kubeAuthInterface: clusterAdminLocalSARGetter,
 		response: authorizationapi.SubjectAccessReviewResponse{
 			Allowed:   false,
 			Reason:    `User "danny" cannot get projects at the cluster scope`,
 			Namespace: "",
 		},
 	}.run(t)
 	subjectAccessReviewTest{
-		description:      "as danny, can I make cluster subject access reviews",
-		clusterInterface: dannyClient.SubjectAccessReviews(),
-		clusterReview:    askCanDannyGetProject,
-		err:              `User "danny" cannot create subjectaccessreviews at the cluster scope`,
+		description:       "as danny, can I make cluster subject access reviews",
+		clusterInterface:  dannyClient.SubjectAccessReviews(),
+		clusterReview:     askCanDannyGetProject,
+		kubeAuthInterface: dannySARGetter,
+		err:               `User "danny" cannot create subjectaccessreviews at the cluster scope`,
+		kubeErr:           `User "danny" cannot create authorization.k8s.io.subjectaccessreviews at the cluster scope`,
```

Comment:
The authorizer's [messages](https://github.com/openshift/origin/blob/master/pkg/authorization/authorizer/messages.go#L34) put the group first (and none of its unit tests provide group info).